spec/models/polling/scheduler_spec.rb

```diff
@@ -0,0 +1,181 @@
+require 'rails_helper'
+
```

Comment:
As discussed: tests for rufus-scheduler functionality that isn't being used by our scheduler class should probably be moved somewhere else (I think they're worth keeping).  I'm not too bothered about leaving them here though; it won't stop me from approving the PR.